src/util-device.c

```diff
@@ -106,6 +109,36 @@ char *LiveGetDeviceName(int number)
 }
 
 /**
+ *  \brief Shorten a device name that is to long
+ *
+ *  \param device name from config and destination for modified
+ *
+ *  \retval None, is added to destination char *newdevname
+ */
+int LiveSafeDeviceName(const char *devname, char *newdevname, size_t destlen)
+{
+    size_t devnamelen = strlen(devname);
+
+    // If we have to shorten the interface name
+    if (devnamelen > MAX_DEVNAME) {
+    
+        // We need 13 chars to do this shortening
+        if (destlen < 13) {
+            return 1;
+        }
+    
+        size_t length;
+        length = strlcpy(newdevname, devname, DEVNAME_CHUNCK);
+        length = strlcat(newdevname, "...", DEVNAME_CHUNCK+3);
```

Comment:
I think 2 dots would be enough to indicate it has been shortened. Gives us one extra byte for useful info :)